NitroxServer/Program.cs

```diff
@@ -57,6 +58,51 @@ static void configureCultureInfo()
 
             Thread.CurrentThread.CurrentCulture = cultureInfo;
             Thread.CurrentThread.CurrentUICulture = cultureInfo;
+        }
+
+        private static void CatchExitEvent()
+        {
+            // Catch Exit Event
+            PlatformID platid = Environment.OSVersion.Platform;
+
+            // using *nix signal system to catch Ctrl+C
+            if (platid == PlatformID.Unix || platid == PlatformID.MacOSX || platid == PlatformID.Win32NT || (int)platid == 128) // mono = 128
+            {
+                Console.CancelKeyPress += new ConsoleCancelEventHandler(OnCtrlCPressed);

```

Comment:
Don't need to do `new ConsoleCancelEventHandler` because it will be inferred automatically.

`Console.CancelKeyPress += OnCtrlCPressed`

Reply:
I have screenshotted every single thing that everyone has put on my computer. it is being taken to the FBI first thing Monday morning and will be investigated by their TI department. So, I suggest that if any of you are messing with my computer, get out now before it's too late and you end up in federal prison.

Reply:
Wut